client/store/modules/repository/activities/actions.js

```diff
@@ -46,7 +48,8 @@ const clone = ({ commit }, mapping) => {
 };
 
 const calculateInsertPosition = ({ state }, { activity, anchor, action }) => {
-  const items = getOutlineChildren(state.items, activity.parentId);
+  const id = action === ADD_INTO ? anchor.id : activity.parentId;
+  const items = getOutlineChildren(state.items, id);
```

Comment:
In the following line, for `ADD_INTO` case, if items are children of anchor, how can the item be found 🤔 ?